modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.server/src/main/scala/edu/gemini/seqexec/web/server/security/authentication.scala

```diff
@@ -72,19 +106,10 @@ object AuthenticationService {
     }
   }
 
-  /**
-    * From the user details it creates a JSON Web Token
-    */
-  def buildToken(u: UserDetails): String =
-    // Given that only this server will need the key we can just use HMAC. 512-bit is the max key size allowed
-    Jwt.encode(JwtClaim(write(u)).issuedNow.expiresIn(3600), AuthenticationConfig.key, JwtAlgorithm.HmacSHA256)
+  // Launcher for the authentication service taking a configuration
+  val authServices: Kleisli[Task, AuthenticationConfig, AuthenticationService] = Kleisli(c =>
+    if (c.devMode) Task.now(AuthenticationService(c))
+    else Task.now(AuthenticationService(c))
+  )
```

Comment:
You're right. `devMode` is used inside the constructor of `AuthenticationService`